pkg/kubelet/prober/prober_manager.go

```diff
@@ -31,6 +32,16 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util/format"
 )
 
+// ProberResults stores the results of a probe as prometheus metrics.
+var ProberResults = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Subsystem: "prober",
+		Name:      "probe_result",
+		Help:      "The result of a liveness or readiness probe for a container.",
+	},
+	[]string{"probe_type", "container_name", "pod_name", "namespace", "pod_uid"},
```

Comment:
not sure if we need both pod_uid and namespace+pod_name each of this should uniquely identify pod.

Reply:
I think in #58827 we decided that it would be good to have both?